ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,29 @@
+ages = [29, 19, 17, 26, 32, 16, 21]
+print(ages)
+print(ages[4:5])
+print(ages[-2:-1])
+print(ages[3:6])
+print(ages[:4])
+
+if 17 in ages:
+  print((ages))
+
+if 42 in ages:
+  print((ages))
+
+ages[2] = 18
+print(ages)
+
+temp = ages[4]
+ages[3] = ages[4]
+temp = ages[3]
+print("Instruction 15: ",ages)
```

Comment:
This didn't swap the fourth and fifth values, it just overwrote the fourth with the fifth:

---

![image](https://user-images.githubusercontent.com/16143771/149258351-1f5b60db-dc1a-422e-a705-b919727ca3bd.png)

---

![image](https://user-images.githubusercontent.com/16143771/149258367-9b4db3ec-418b-4138-a760-fa7ed39a6c36.png)